contrib/wireshark/wireshark.pre-install

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "g _wireshark -" | \
+    /usr/bin/systemd-sysusers --replace /usr/lib/sysusers.d/wireshark.conf - || :
```

Comment:
this sux but just like in qemu i assume there's no other way to have a group on an owned file like this